### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by siddharth-sahoo**
Fixing errors in signal-to-noise ratio explanation for batching frequency and campaign variables section.

### CODE REVIEW

**Comment by keke123** [comment]
Changes LGTM. Fewer conversion events and more frequent reports should lead to a lower signal-to-noise ratio

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ batching frequency is how often you process aggregatable reports.
 A report that is scheduled for aggregation more frequently (e.g. each hour) will
```
Comment (nancymic2) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:234]:
```suggestion
A report that is scheduled for aggregation more frequently (e.g. each hour) will
have fewer conversion events included than the same report with a less frequent
aggregation schedule (e.g. each week). As a result, the hourly report will include more noise.```

**Review by nancymic2 — COMMENTED** [review]
Left suggestions

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 2]
```diff
@@ -276,7 +276,7 @@ single-touch attributable conversions to the closest power of 10: 10, 100,
 1. Click **Simulate**.
 1. Observe that the noise ratios are now higher: this is because when you
    have fewer conversions per bucket, signal-to-noise ratios are likely to be
-   higher.
+   lower.
```
[the remark this answers is not in the document]
Comment (siddharth-sahoo) [reply site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md]: Done!

**Review by nancymic2 — APPROVED** [review]

**Comment by nancymic2** [comment]
@rachelandrew I'm coming back as not having a valid CLA  (?)

**Comment by rachelandrew** [comment]
> @rachelandrew I'm coming back as not having a valid CLA (?)

You have made the commits using your gmail address it looks like.

**Comment by nancymic2** [comment]
> > @rachelandrew I'm coming back as not having a valid CLA (?)
> 
> You have made the commits using your gmail address it looks like.

I saw that but no idea how. thanks will look into it

**Pull request merged**